src/qt/walletmodel.h

```diff
@@ -190,7 +191,7 @@ class WalletModel : public QObject
     UnlockContext requestUnlock();
 
     bool getPubKey(const CKeyID &address, CPubKey& vchPubKeyOut) const;
-    bool havePrivKey(const CKeyID &address) const;
+    isminetype isMine(const CTxDestination &dest) const;
```

Comment:
`const CTxDestination& dest`.

Reply:
Another option, just for the record, that doesn't require to include `script/ismine.h`, is `bool isSpendable(const CKeyID& address)`.

Reply:
I don't see `bool isSpendable(const CKeyID& address)` anywhere. And the point is that we want to be able to check the spendability of general addresses, not only `CKeyID`s.